src/main/java/mate/academy/intro/dto/UserLoginRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.intro.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+
+public record UserLoginRequestDto(
+        @NotEmpty
```

Comment:
for String validation use `@NotBlank`
check and fix other places
```suggestion
        @NotBlank
```